server/src/test/java/io/crate/analyze/GroupByAnalyzerTest.java

```diff
@@ -118,6 +121,13 @@ public void testGroupByWithDistinctAggregation() throws Exception {
         analyze("select count(DISTINCT name) from sys.nodes group by 1");
     }
 
+    @Test
+    public void test_group_by_unknown_column() {
+        Assertions.assertThrows(IllegalArgumentException.class,
+                                () -> analyze("select max(age) from foo.users group by details_ignored['lol']"),
+                                "Using column type ignored 'details_ignored['lol']' for grouping or aggregations is not allowed");
+    }
```

Comment:
We could probably just go with the analyzer test and remove the integration test.

Reply:
Yep agreed